### ISSUE TITLE
Addon is not resetting the new card count for my decks. 

### ISSUE THREAD

**Issue by burnhamnate** [BSCrumpton/Deadline2#34]
I just reinstalled the addon, and when I click on process deadlines, it shows up with the correct cards/ day ratio to see all the new cards by a certain day. When I close that box and go back to my main screen, the new card count does not change on the deck. I have it set to the normal 20new/200 reviews per day. I have also tried 9999new/9999review and that does not work either.

**Comment by burnhamnate** [comment]
I haven't used this addon since the new scheduler is out. Could that be the issue? Please let me know if you can fix this. I love this addon so much.

**Comment by BSCrumpton** [comment]
@burnhamnate that's correct. Sorry, I haven't made an effort to make this addon compatible with the newest scheduler yet.

**Comment by mistervelez** [comment]
Any news on if there will be an update to make the addon compatible with FSRS?
